src/search.c

```diff
@@ -153,7 +150,83 @@ static void free_bundle_result_data(void *data)
 	free(br);
 }
 
-static void print_final_results(void)
+/* unmark all bundle_results in bl as seen */
+static void unsee_bundle_results(struct list *bl)
```

Comment:
s/unsee/unmark?

Reply:
I appreciate the review, but this is the commit from #399. Can you review that commit there? This PR is based on that one.

Also, sure I suppose. I kinda like unsee though since we are tracking if we've seen the bundle. The mark is only to tell us if we've seen it. So if the function name is unmark_* what are we unmarking?